worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -912,16 +916,16 @@ namespace RTC
 		}
 
 		// Process the packet.
-		if (this->rtpStream->ReceivePacket(packet))
+		if (this->rtpStream->ReceivePacket(sharedPacket ? nullptr : packet, sharedPacket))
 		{
 			if (this->rtpSeqManager.GetMaxOutput() == packet->GetSequenceNumber())
 				this->lastSentPacketHasMarker = packet->HasMarker();
 
 			// Send the packet.
-			this->listener->OnConsumerSendRtpPacket(this, packet.get());
+			this->listener->OnConsumerSendRtpPacket(this, packet);
 
 			// May emit 'trace' event.
-			EmitTraceEventRtpAndKeyFrameTypes(packet.get());
+			EmitTraceEventRtpAndKeyFrameTypes(packet);
```

Comment:
No need to change this as well in other Consumer classes such as DirectConsumer?

Reply:
It's changed in every consumer: Simple, Simulcast, Svc, Pipe

Reply:
@ibc?

Reply:
So we were not using it in DirectConsumer?